docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -65,7 +75,7 @@ yarn add aws-amplify @aws-amplify/ui-react@3.x
 
 ### Update and usage
 
-The following breaking changes where introduced in `@aws-amplify/ui-react` major version `3.0`:
+The following breaking changes were introduced in `@aws-amplify/ui-react` major version `3.0`:
```

Comment:
I think this needs to be removed since it is above?

Reply:
good catch, will update

Reply:
Oh, no actually, this was just a typo fix on the 3.0 release's migration instructions. It's not a duplicate.